src/main/webapp/script.js

```diff
@@ -528,7 +528,7 @@ function storeRepBooleanAndZipcodeAndRedirect() {
 
 //function for loading indicator for rep list page
 function loadingIndicator() {
-  myVar = setTimeout(getRepList, 3000);
+  myVar = setTimeout(getRepList, 125);
```

Comment:
what is the unit for 125? use a constant to give it a name like TIMEOUT_MILLIS = 125;